.github/workflows/build-wheel.yml

```diff
@@ -102,53 +116,20 @@ jobs:
           python -c "import mlflow; print(mlflow.__version__)"
 
       # Anyone with read access can download the uploaded wheel on GitHub.
-      - name: Store wheel
+      - name: Upload wheel
         uses: actions/upload-artifact@v4
-        if: github.event_name == 'push'
+        if: github.event_name == 'workflow_dispatch'
+        id: upload-wheel
         with:
           name: ${{ steps.build-dist.outputs.wheel-name }}
           path: ${{ steps.build-dist.outputs.wheel-path }}
+          retention-days: 7
+          if-no-files-found: error
 
-      - name: Remove old wheels
```

Comment:
Removed this step to let GitHub delete the artifact when it expires.

Reply:
Does this job create unique artifact per run? An MLflow wheel is roughly 5MB, then 500MB quota means 100 runs until hitting the limit. Considering the amount of commit we push to PRs daily, we can easily breach the quota in 7 days? Wondering if we want to make it retention days shorter or auto-delete for PR runs🤔

Reply:
Once https://github.com/mlflow/mlflow/pull/13356/files#r1793402256 is applied, this workflow only uploads when manually triggered :)